packages/core/src/file_app_store.ts

```diff
@@ -72,7 +72,7 @@ export class FileAppStore {
 
     buildApp(
       cwd,
-      { watch: true, uglify: this.options.uglify },
+      { watch: this.options.noWatch || true, uglify: this.options.uglify },
```

Comment:
Random passerby here, trying to find my way to the v8/isolate code... was `|| true` kept here intentionally @michaeldwan ?